### ISSUE TITLE
ydotoold steals focus

### ISSUE THREAD

**Issue by egolus** [ReimuNotMoe/ydotool#59]
I'm using rofi-pass on arch linux with the ydotool patch from [aur](https://aur.archlinux.org/packages/rofi-pass-ydotool-git/).

`pacman -Qi ydotool-git` reports `Version: v0.1.8.r29.g7764122-3`

When I open rofi-pass, it loses focus after a short time (a second or so). Also when holding a key down in a terminal the key doesn't repeat or stops after a short time. If I stop ydotoold, the key repeats normally.

can I fix this with some config change or is this a problem with ydotoold itself?

**Comment by ClassicOldSong** [comment]
Can you try reproduce this issue in other apps?

**Comment by egolus** [comment]
my standard terminal is kitty. I just tested holding down the key in termite and in Gnvim and both times the input stops after a few characters. After killing `ydotoold` I can hold down a key and it keeps putting characters into the window.

with rofi my understanding is that it loses focus if the window that was focused before is a terminal, I tested both kitty and termite

what I haven't said, yet is that I'm using sway as window manager
